src/blueprint/stringify/response-to-string.ts

```diff
@@ -15,7 +15,12 @@ export default class ResponseToString {
 
   private stringifyResponse(nestLevel: number = 0, response: ApiBlueprint.Response): string {
     let result = common.createListItem(`Response ${response.identifier} (${response.mediaType})`, nestLevel);
-    result += common.createSentence(response.value, true, 3);
+    result += common.createListItem(`Body`, 2);
```

Comment:
result += common.createListItem(`Body`, nestLevel + 2);

also do same for lines bellow